pnpm-workspace.yaml

```diff
@@ -4,3 +4,26 @@ packages:
   - e2e-tests
   - tools/codemod
   - docs/storybook
+
+catalog:
+  # iTwin.js core packages
```

Comment:
Have you considered using named catalogs instead of a comment? E.g:
```yaml
catalogs:
  itwinjs-core:
    '@itwin/appui-abstract': rc
    ...
```

Reply:
I did, but there's no use-case for named catalogs yet.